crm_claim_ref_smartbutton/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# (c) 2015 Pedro M. Baeza
+# License AGPL-3 - See LICENSE file on root folder for details
+##############################################################################
+
+{
+    "name": "Smart-button for referenced claims",
+    "version": "1.0",
+    "depends": [
+        "crm_claim",
+    ],
+    "author": "AvanzOSC, "
+              "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/crm",
+    "contributors": [
+        "Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>",
+        "Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>",
+    ],
+    "category": "Customer Relationship Management",
+    "installable": True,
+    "pre_init_hook": "pre_init_hook",
```

Comment:
Can you draft the proposition with the fake model? Take into account that this module can cover any linkable model in claims.